insights/mappers/certificates_enddate.py

```diff
@@ -67,3 +73,22 @@ def get_expiration_date(self, path):
                     if self.data and path in self.data else None)
         except:
             raise Exception("Unable to parse the expiration data of %s" % path)
+
+    def expiration_date(self, path):
+        """This will return a namedtuple(['str', 'datetime']) contains the
+        expiration date in string and datetime format. If the expiration date
+        is unparsable, the ExpirationDate.datetime should be None.
+
+        Args:
+            path(str): the certificate file path
+
+        Returns:
+            A ExpirationDate for available path. None otherwise.
```

Comment:
Actually, Not "None" here, but an ExpirationDate only the `datetime` is None, the "RAW" format of the datestamp `.str` is still returned.